src/Control/Algebra.hs

```diff
@@ -137,7 +137,7 @@ instance Monoid w => Algebra (Writer w) ((,) w) where
 instance Algebra sig m => Algebra (Error e :+: sig) (Except.ExceptT e m) where
   alg (L (L (Throw e)))     = Except.throwE e
   alg (L (R (Catch m h k))) = Except.catchE m h >>= k
-  alg (R other)             = Except.ExceptT $ alg (handle (Right ()) (either (pure . Left) Except.runExceptT) other)
+  alg (R other)             = Except.ExceptT $ alg (thread (Right ()) (either (pure . Left) Except.runExceptT) other)
```

Comment:
Unlike #300, I haven’t opted to introduce `handle ctx hdl` as a synonym for `alg . thread ctx hdl` because I have some reservations about `handleCoercible` & `handleIdentity` potentially being error-prone (which I intend to explore in a separate PR).